wme_street_to_river_plus.user.js

```diff
@@ -482,7 +482,23 @@ function streetToRiver_init() {
       // 2014-10-08: Add new Landmark to Waze Editor
       var riverLandmark_o = new wazeActionAddLandmark(riverLandmark);
       W.model.actionManager.add(riverLandmark_o);
-      W.selectionManager.setSelectedModels([riverLandmark]);
+      try {
+        W.selectionManager.setSelectedModels([riverLandmark]);
+      } catch (err) {
+        // Ignore error:
+          // Uncaught TypeError: Cannot read properties of undefined (reading 'children')
+          // at Object.WMETB_FPnewSelectionAvailable (FancyPermalink.min.js:216:184)
+          // at v (third_party-45fe9aba9d649e1fe91a.js.gz:2:1169484)
```

Comment:
I guess, 2 lines should be enough, the rest of callstack can be removed

Reply:
Sounds reasonable, done